src/apis/Dimensions/index.js

```diff
@@ -44,6 +46,26 @@ export default class Dimensions {
       scale: win.devicePixelRatio || 1,
       width: win.screen.width
     };
+
+    if (canUseDOM) {
+      const change: window.Event = new window.Event(EVENT_CHANGE);
```

Comment:
I saw that there's no kind of an event emitter in dependencies, so custom DOM event is the only choice for me at that time. I also checked it's compatibility, it doesn't look bad. Do you have a better idea? What about adding `eventemitter3`?